sklearn/model_selection/tests/test_search.py

```diff
@@ -1,69 +1,69 @@
 from sklearn.datasets import make_blobs
+from sklearn.datasets import make_classification
 from sklearn.datasets import make_multilabel_classification
-
+from sklearn.externals.six.moves import cStringIO as StringIO
+from sklearn.externals.six.moves import xrange
+from sklearn.externals.six.moves import zip
+from sklearn.manifold.t_sne import TSNE, trustworthiness
+from sklearn.metrics import f1_score
+from sklearn.metrics import make_scorer
+from sklearn.metrics import roc_auc_score
+from sklearn.metrics import silhouette_score
+from sklearn.model_selection import GridSearchCV, UnsupervisedGridSearch
 from sklearn.model_selection import KFold
-from sklearn.model_selection import StratifiedKFold
-from sklearn.model_selection import StratifiedShuffleSplit
-from sklearn.model_selection import LeaveOneLabelOut
-from sklearn.model_selection import LeavePLabelOut
 from sklearn.model_selection import LabelKFold
 from sklearn.model_selection import LabelShuffleSplit
-from sklearn.model_selection import GridSearchCV
-from sklearn.model_selection import RandomizedSearchCV
+from sklearn.model_selection import LeaveOneLabelOut
+from sklearn.model_selection import LeavePLabelOut
 from sklearn.model_selection import ParameterGrid
 from sklearn.model_selection import ParameterSampler
-
-# TODO Import from sklearn.exceptions once merged.
-from sklearn.base import ChangedBehaviorWarning
+from sklearn.model_selection import RandomizedSearchCV, UnsupervisedRandomizedSearch
+from sklearn.model_selection import StratifiedKFold
+from sklearn.model_selection import StratifiedShuffleSplit
 from sklearn.model_selection._validation import FitFailedWarning
-
-from sklearn.svm import LinearSVC, SVC
-from sklearn.tree import DecisionTreeRegressor
-from sklearn.tree import DecisionTreeClassifier
-from sklearn.cluster import KMeans
 from sklearn.neighbors import KernelDensity
-from sklearn.metrics import f1_score
-from sklearn.metrics import make_scorer
-from sklearn.metrics import roc_auc_score
-from sklearn.preprocessing import Imputer
 from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import Imputer
+from sklearn.svm import LinearSVC, SVC
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.tree import DecisionTreeRegressor
+from sklearn.utils.fixes import sp_version
+from sklearn.utils.mocking import CheckingClassifier, MockDataFrame
+from sklearn.utils.testing import assert_almost_equal
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_false, assert_true
+from sklearn.utils.testing import assert_no_warnings
+from sklearn.utils.testing import assert_not_equal
+from sklearn.utils.testing import assert_raise_message
+from sklearn.utils.testing import assert_raises
+from sklearn.utils.testing import assert_warns
+from sklearn.utils.testing import ignore_warnings
```

Comment:
Same comment as the previous one... Could you kindly avoid making changes not related to the PR. (This change is indeed welcome as it cleans up, but will blow up your diff and *may* distract reviewers.)

(Hope you don't mistake the aggressive nitpicking.)